remote-wallet/src/remote_wallet.rs

```diff
@@ -205,6 +237,24 @@ impl RemoteWalletManager {
         }
         false
     }
+
+    pub fn has_trezor_client(&self) -> bool {
+        self.trezor_client.is_some()
+    }
+
+    pub fn get_trezor_client_clone(&self) -> Option<Rc<RefCell<Trezor>>> {
+        self.trezor_client.as_ref().map(Rc::clone)
+    }
+
+    pub fn get_trezor_wallet(
```

Comment:
ideally we don't introduce implementation specific interfaces like this. `get_ledger()` needs deprecated